CASMINST-4866 Layout Changes for install/README.md (#1934)
Cray-HPE/docs-csm: upgrade/1.2/resource_material/k8s/worker-reference.md

   cray cps deployment update --nodes "ncn-w002"
   ```

1. Confirm the CFS `configurationStatus` for **all** worker nodes before shutting down this worker node. If the state is `pending`,
   the administrator may want to tail the logs of the CFS pod running on that node to watch the job finish

>    before rebooting this node. If the state is `failed` for this node, then you will know that the failed CFS job state

Reviewer: ```suggestion
   before rebooting this node. If the state is `failed` for this node, then this indicates that the failed CFS job state
```

>    preceded this worker rebuild, and that can be addressed independent of rebuilding this worker.

Reviewer: ```suggestion
   preceded this worker rebuild, and that it can be addressed independent of rebuilding this worker.
```
